utils/install-app.js

```diff
@@ -139,6 +141,78 @@ function addToServer(appDir, installDir) {
   }
 }
 
+function loadRecognizers(appDir, appId, appVers) {
+  let recognizers;
+  let configRecognizers;
+  let configLocation;
+
+  try { // Get recognizers in a plugin's appDir/config/xxx location
+    recognizers = JSON.parse(fs.readFileSync(path.join(appDir, "config/recognizers", appId))).recognizers;
+    const keys = Object.keys(recognizers)
+    for (const key of keys) { // Add metadata for plugin version & plugin identifier of origin (though objects don't have to be plugin specific)
+      recognizers[key].pluginVersion = appVers;
+      recognizers[key].pluginIdentifier = appId;
+    }
+    logger.debug('ZWED0294I', appId); //logger.debug("Found recognizers for '" + appId + "'");
+  } catch (e) {
+    logger.debug('ZWED0295I', (path.join(appDir, "config/recognizers", appId)), appId); //logger.debug("Could not find recognizers in '" + (path.join(appDir, "config/recognizers", appId)) + "'");
+  }
+
+  try { // Get pre-existing recognizers in config, if any
+    configLocation = path.join(process.env.ZLUX_CONFIG_FILE.substring(1), "../../ZLUX/pluginStorage/org.zowe.zlux.ng2desktop/");
```

Comment:
I'm pretty sure this env var is NOT guaranteed to exist.
when the server runs, INSTANCE_DIR is guaranteed.
When run from install-app.sh, it should be also, but it look like it isnt... easy fix? It seems like a minor refactor is needed because when INSTANCE_DIR doesnt exist, the user at least has provided the server.json location: 
https://github.com/zowe/zlux-app-server/blob/master/bin/install-app.sh#L74
... it ends up as "zluxConfig" in the top of this file.